### ISSUE TITLE
Pinout issue

### ISSUE THREAD

**Issue by MrMojo4Risin** [pkucmus/3dprinting-docs#1]
I am trying to follow your method but get a lot of random moves on all axes. Did you cut out the pin from the bottom where you connected the jumper wire?

Thanks

**Comment by pkucmus** [comment]
Did no modification to the board it self, if that's what you are asking. Maybe the software is trying to use the pins for something. Is the board set correctly in Marlin?

**Comment by MrMojo4Risin** [comment]
Yes the marlin seems to be ok, all other components works as it should be . I was wondering if you chipped off bottom pin from the driver as it is connected by the jump wire. Wanted to ask before getting other drivers to test. Thanks for the answer
